### PR TITLE
Fix(CWC) - exists is not a function

### PR DESCRIPTION
**Pull request by justinkook**
- removes `secp256k1` and replaced with bitcoreLib point
- increase test coverage to all functions.
- remove `ethereumjs-tx` has dependencies on `secp256k1`
- remove `ethereumjs-util` has dependencies on `secp256k1`
- remove `elliptic` unused
- remove `bcrypt` unused
- replaces `ethereumjs` functions with `ethers`

### CODE REVIEW

File: packages/crypto-wallet-core/tests/address.ts [hunk 1]
```diff
@@ -17,4 +17,25 @@ describe('Address Derivation', () => {
     const expectedAddress = '0x9dbfE221A6EEa27a0e2f52961B339e95426931F9';
     expect(address).to.equal(expectedAddress);
   });
+
+  it('should be able to generate a valid ETH address, privKey, pubKey', () => {
```
Comment (matiu) [inline packages/crypto-wallet-core/tests/address.ts]: 😍

File: packages/crypto-wallet-core/tests/transactions.ts [hunk 2]
```diff
@@ -20,7 +20,7 @@ describe('Transaction Creation', () => {
...
+      '0xf9014c808504a817c800809437d7b3bbd88efde6a93cf74d2f5b0385d3e3b08a870dd764300b8000b90124b6b4af05000000000000000000000000000000000000000000000000000dd764300b800000000000000000000000000000000000000000000000000000000004a817c8000000000000000000000000000000000000000000000000000000016ada606a26050bb49a5a8228599e0dd48c1368abd36f4f14d2b74a015b2d168dbcab0773ce399393220df874bb22ca961f351e038acd2ba5cc8c764385c9f23707622cc435000000000000000000000000000000000000000000000000000000000000001c7e247d684a635813267b10a63f7f3ba88b28ca2790c909110b28236cf1b9bba03451e83d5834189f28d4c77802fc76b7c760a42bc8bebf8dd15e6ead146805630000000000000000000000000000000000000000000000000000000000000000';
```
Comment (matiu) [inline packages/crypto-wallet-core/tests/transactions.ts]: what this is changed? no only  the beginning `0x`, but also the tail bytes? Is there a missing checksum or something?

Comment (matiu) [reply packages/crypto-wallet-core/tests/transactions.ts]: How this will affect upstream libraries?

Comment (micahriggan) [reply packages/crypto-wallet-core/tests/transactions.ts]: This could be from us not putting chain code: 1 on the transaction

Comment (matiu) [reply packages/crypto-wallet-core/tests/transactions.ts]: Would this difference create problems in consuming libraries?

Comment (justinkook) [reply packages/crypto-wallet-core/tests/transactions.ts]: `1c8080` was actually `chainId: 28`. If we pass in `chainId: 1` it results in `018080`

**Pull request merged**